.github/workflows/agw-workflow.yml

```diff
@@ -0,0 +1,311 @@
+          SLACK_COLOR: "#FF0000"
+  mme-clang-tidy:
+    runs-on: ubuntu-latest
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+      BRANCH: "${{ github.base_ref }}"
+      REVISION: "${{ github.sha }}"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run mme-clang-tidy
+        uses: nick-invision/retry@v2
+        # yamllint disable rule:line-length
+        with:
+          command: |
+            cd ${{ env.MAGMA_ROOT }}/lte/gateway/docker/mme
+            docker build --tag magma-mme-build --file Dockerfile.ubuntu20.04 ../../../../
+            docker run --env BRANCH=${{ env.BRANCH}} --env REVISION=${{ env.REVISION }} --volume ${{ env.MAGMA_ROOT }}:/magma --interactive magma-mme-build:latest /bin/bash --cpu-shares 'cd /magma/lte/gateway;make clang_tidy_oai_upload'
+          timeout_minutes: 20
+      # yamllint enable
+      # Notify ci channel when failing
+      # Plugin info: https://github.com/marketplace/actions/slack-notify
+      - name: Notify failure to slack
+        if: failure() && github.ref == 'refs/heads/master'
+        uses: rtCamp/action-slack-notify@v2.0.2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_CI }}
+          SLACK_TITLE: "Github action mme-clang-tidy failed"
+          SLACK_USERNAME: "AGW workflow"
+          SLACK_ICON_EMOJI: ":boom:"
+          SLACK_COLOR: "#FF0000"
+  mme-clang-warnings:
+    runs-on: ubuntu-latest
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+      BRANCH: "${{ github.base_ref }}"
+      REVISION: "${{ github.sha }}"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run mme-clang-warnings
+        uses: nick-invision/retry@v2
+        # yamllint disable rule:line-length
+        with:
+          command: |
+            cd ${{ env.MAGMA_ROOT }}/lte/gateway/docker/mme
+            docker build --tag magma-mme-build --file Dockerfile.ubuntu20.04 ../../../../
+            docker run --env BRANCH=${{ env.BRANCH }} --env REVISION=${{ env.REVISION }} --volume ${{ env.MAGMA_ROOT }}:/magma --volume ${{ env.MAGMA_ROOT }}/lte/gateway/configs:/etc/magma --interactive magma-mme-build:latest /bin/bash --cpu-shares "cd /magma/lte/gateway;make clang_warning_oai_upload"
+          timeout_minutes: 20
+      # yamllint enable
+      # Notify ci channel when failing
+      # Plugin info: https://github.com/marketplace/actions/slack-notify
+      - name: Notify failure to slack
+        if: failure() && github.ref == 'refs/heads/master'
+        uses: rtCamp/action-slack-notify@v2.0.2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_CI }}
+          SLACK_TITLE: "Github action mme-clang-warnings failed"
+          SLACK_USERNAME: "AGW workflow"
+          SLACK_ICON_EMOJI: ":boom:"
+          SLACK_COLOR: "#FF0000"
+  c-cpp-codecov:
+    runs-on: ubuntu-latest
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run codecov
+        uses: nick-invision/retry@v2
```

Comment:
Let's not use this module, just use a regular run command

Reply:
sure!